quickwit/quickwit-janitor/src/actors/delete_task_pipeline.rs

```diff
@@ -406,6 +406,8 @@ mod tests {
             .find(|split| split.split_state == SplitState::Published)
             .unwrap();
         assert_eq!(published_split.split_metadata.delete_opstamp, 1);
+        universe.assert_quit().await;
```

Comment:
We should probably have an accessor to the Universe in `test_sandbox` and avoid having two universes in the same unit test.

(And eventually rework TestSandbox, I am very unhappy with its current state)